feat: keyboard shortcut added for opening editor modal

therealrinku opened the pull request:
resolves: #1923 
_**shift + enter**_ keyboard shortcut added for opening editor modal

rathboma commented:
@azmy60 wanna review this? If good, approve and merge!

azmy60 commented on apps/studio/src/components/tableview/TableTable.vue:
```diff
@@ -974,14 +975,20 @@ export default Vue.extend({
+      const range: RangeComponent = _.last(this.tabulator.getRanges())
+      const cell = range.getCells().flat()[0];
+      if (this.isEditorMenuDisabled(cell)) return
+      this.$refs.editorModal.openModal(cell.getValue(), undefined, cell)
```
Would you be able to add a FIXME note here so we can avoid calling child methods directly like this? Normally, it should be done by calling an event using `this.$modal.show(modalName)` or `this.$trigger(AppEvent.something)`.

azmy60 commented on apps/studio/src/components/tableview/TableTable.vue:
```diff
@@ -411,6 +411,7 @@ export default Vue.extend({
       //   const focusingTable = this.tabulator.element.contains(document.activeElement)
       //   if (!focusingTable) this.page--
       // }
+      result['alt+m'] = this.openEditorMenuByShortcut.bind(this)
```
Should we press `shift+enter` (DBeaver uses this)? Or anything similar with pressing `enter`? I usually look at other tools to decide. I'm just not too familiar with `alt+m` though I feel like it might works or I might be too nitpicky. Feel free to use anything that's best for you.

azmy60 reviewed (changes requested):
Just a couple of notes, and we're good :)

therealrinku commented:
hi @azmy60 ! thanks for the review!
there was not any specific reason to add **alt + m** shortcut. I agree, that it makes more sense to use **shift + enter* so I updated it to use **shift + enter** and added FIXME note as well :)

azmy60 reviewed (approved):
Perfect! Thanks so much.
